src/PartitionMgr/test/change_eval_method.tcl

```diff
@@ -0,0 +1,17 @@
+# Check if changing evaluation methods results in different answers
```

Comment:
Why not add this to regression_tests.tcl?  I don't understand your "Test case included for when unit testing is available." comment.

Reply:
at the moment the partitioning tools are not compiled in by default, so this test wont work until those are compiled in as well (https://openroad.atlassian.net/browse/OR-947)

Reply:
How do the other tests run then?

Reply:
The ones that are currently being used? they only report the graph information, but does not attempt to run the partitioning or clustering.